network/bluetooth_att.ksy

```diff
@@ -0,0 +1,117 @@
+meta:
+  id: bluetooth_att
+  title: Bluetooth ATT (IEEE 802.15.1)
+  xref:
+    ieee: 802.15.1
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: le
+doc: |
+  Bluetooth ATT is a part of the Bluetooth specification.
+  Attribute Profile, page 2288
+doc-ref: https://www.bluetooth.org/docman/handlers/downloaddoc.ashx?doc_id=457080
+
+seq:
+  - id: opcode
+    type: u1
+    enum: att_opcode
+    doc: page 2330
+  - id: parameters
+    type:
+      switch-on: opcode
+      cases:
+        'att_opcode::error_response': error_response
+        'att_opcode::find_information_request': read_by_type_request
+        'att_opcode::find_information_response': find_information_response
+        'att_opcode::read_by_type_request': read_by_type_request
+        'att_opcode::read_by_type_response': read_by_type_response
+        'att_opcode::read_by_group_type_request': read_by_type_request
+        'att_opcode::read_by_group_type_response': read_by_type_response
+        'att_opcode::write_request': write_request
+        'att_opcode::handle_value_notification': write_request
+
+enums:
+  # 3.4.8 Attribute Opcode summary, page 2330
+  att_opcode:
```

Comment:
Should it be decomposed, splitting the LSB into a direction and the rest into opcode?

Reply:
There's no "direction" in the ATT part. Please clarify.

Reply:
there is no, but you can try to be creative. I see the enums have internal structure. The LSB (even-odd) codes direction, the rest of bits code opcode.

Reply:
The point of transcribing specification is not really about being "creative", but following the prescribed doc => so I would strongly recommend against inventing stuff unless there's a very good reason behind that.

Reply:
I think we can postpone creative parts to later commits.